src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderList.jsx

```diff
@@ -125,7 +125,12 @@ export default function ProviderList({
             />
           )}
           {currentlyShownProvidersList.length > 0 && (
-            <>
+            <fieldset>
+              <legend className="sr-only">
+                Choose {/^[aeiou]/i.test(typeOfCareName) ? 'an' : 'a'}{' '}
```

Comment:
nit: Ha, channeling my English teacher, we should probably use something like https://www.npmjs.com/package/indefinite to take care of all the exceptions like "an hour". I'm kidding of course but we also do have the unused utility https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/utils/formatters.js#L47.

Reply:
I was looking for that utility but couldn't find it.